datahub-web-react/src/app/shared/tags/AddTagTermModal.tsx

```diff
@@ -74,49 +68,56 @@ export default function AddTagTermModal({
     entitySubresource,
     type = EntityType.Tag,
 }: AddTagModalProps) {
-    const [getAutoCompleteResults, { loading, data: suggestionsData }] = useGetAutoCompleteMultipleResultsLazyQuery({
-        fetchPolicy: 'no-cache',
-    });
     const [inputValue, setInputValue] = useState('');
     const [selectedValue, setSelectedValue] = useState('');
     const [showCreateModal, setShowCreateModal] = useState(false);
     const [disableAdd, setDisableAdd] = useState(false);
     const entityRegistry = useEntityRegistry();
     const [addTagMutation] = useAddTagMutation();
     const [addTermMutation] = useAddTermMutation();
+    const [tagSearch, { data: tagSearchData }] = useGetSearchResultsLazyQuery();
+    const tagSearchResults = tagSearchData?.search?.searchResults || [];
 
-    const autoComplete = (query: string) => {
-        if (query && query !== '') {
-            getAutoCompleteResults({
+    const handleSearch = (text: string) => {
+        if (text.length > 2) {
```

Comment:
On thinking about this again.. some people have tags and terms of length = 0

Can we update this to be text.length > 0?

Reply:
Fixed and commited.